02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,86 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+<meta charset="utf-8"/>
+<title>
+Me
+</title>
 </head>
-<body>
+<body style="background-color:PaleGreen;">
+<div style="text-align: center;">
+<h1>
+Jamie Coe
+</h1>
+<h3>
+Austin, TX
+</h3>
+<img src="https://scontent-dft4-1.xx.fbcdn.net/v/t1.0-1/p160x160/14457492_10154548329177733_8641882206166032071_n.jpg?oh=32aa5f26d111f8fe8fc1678956c02c37&oe=58616B55" alt="facebookimage"/>
+<p>
+I'm never very good at describing myself. I tend to spend a lot of time
+  in the background. I love playing video games and learning new things.
+</p>
+</div>
+<p style="color:Salmon">Interests</p>
+<ul>
+<li style="font-family:Futura">Reading</li>
+<li style="font-family:Futura">Writing</li>
+<li style="font-family:Futura">Learning to code</li>
+</ul>
+<p style="color:Salmon">Jobs</p>
+<ol>
+<li style="font-family:Futura">Medical Laboratory Scientist</li>
+<li style="font-family:Futura">Aspiring Web developer</li>
+</ol>
+<p style="color:Salmon">Schools attended</p>
+<ol>
+<li style="font-family:Futura">River Valley High School</li>
+<li style="font-family:Futura">University of Rio Grande (BS)</li>
+<li style="font-size:18px; font-family:Futura">Austin Coding Academy (Current)</li>
+</ol>
+<p style="color:Salmon">Favorite Quote</p>
+<ul>
+<li style="font-family:Futura">"Anyone who has never made a mistake has never tried anything new."</li>
+</ul>
+<table>
+<thead>
+  <tr>
+<th colspan="2" style="color:Salmon;">Favorite Shows</th>
+  </tr>
+</thead>
+  <tbody>
+  <tr>
+  <td>
+    <a href="http://www.sho.com/penny-dreadful">
+    <img src="https://images-na.ssl-images-amazon.com/images/I/618dzoJCEtL._AC_UL320_SR244,320_.jpg"
+    alt="penny dreadful"
+    style="width:128px;height:128px;border:0;">
+    </a>
+  </td>
+<td>
+<a href="http://www.hbo.com/westworld">
+  <img src="https://pbs.twimg.com/profile_images/771377480850628608/ZMA4LhKD.jpg"
+  alt="westworld"
+  style="width:128px;height:128px;border:0;">
+  </a>
+  </td>
+  </tr>
+<tr>
+  <td>
+<a href="http://www.hbo.com/game-of-thrones">
+    <img src="https://pbs.twimg.com/profile_images/702545332475981824/Mg7TpOaw.jpg"
+    alt="game of thrones"
+    style="width:128px;height:128px;border:0;">
+    </a>
+  </td>
+  <td>
+<a href="http://www.amc.com/shows/halt-and-catch-fire">
+    <img src="https://pmcdeadline2.files.wordpress.com/2015/02/halt-and-catch-fire-logo.png?w=446&h=299&crop=1"
+    alt="halt and catch fire"
+    style="width:128px;height:128px;border:0;">
+    </a>
+  </td>
+</tr>
+  </tbody>
+</table>
```

Comment:
The indentation is missing here or inconsistent. Did the change in Atom from Tab to spaces mess that up? Try to make this consistent for all elements.